src/test/scala/nl.knaw.dans.easy.deposit/servlets/NotFoundSpec.scala

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.deposit.servlets
+
+import better.files.File
+import nl.knaw.dans.easy.deposit.PidRequesterComponent.PidRequester
+import nl.knaw.dans.easy.deposit.authentication.AuthenticationMocker._
+import nl.knaw.dans.easy.deposit.authentication.AuthenticationProvider
+import nl.knaw.dans.easy.deposit.docs.DepositInfo
+import nl.knaw.dans.easy.deposit.{ EasyDepositApiApp, TestSupportFixture }
+import nl.knaw.dans.lib.error._
+import org.eclipse.jetty.http.HttpStatus._
+import org.scalamock.scalatest.MockFactory
+import org.scalatra.test.scalatest.ScalatraSuite
+
+class NotFoundSpec extends TestSupportFixture with ServletFixture with ScalatraSuite with MockFactory {
```

Comment:
Rather than collecting these tests here, I would rather put them with the other tests for the specific servlet. Is there a special reason for putting them in a separate file?

Reply:
Most servlet test are about one servlet, so they are grouped by themes.